Lib/test/certdata/make_ssl_certs.py

```diff
@@ -219,14 +223,22 @@ def make_ca():
     shutil.copy('capath/ceff1710.0', 'capath/b1930218.0')
 
 
-def print_cert(path):
+def write_cert_reference(path):
     import _ssl
-    pprint.pprint(_ssl._test_decode_cert(path))
+    refdata = pprint.pformat(_ssl._test_decode_cert(path))
+    print(refdata)
+    with open(path + '.reference', 'w') as f:
+        f.write(refdata)
```

Comment:
When I ran the script, the reference files were generated without trailing newlines. In this PR they have the newlines; I assume some tool added them as you added the files to Git.

Let's have the script print the newlines:
```suggestion
        print(refdata, file=f)
```